tox.ini

```
@@ -45,7 +45,7 @@ skip_install = true
 commands =
     python setup.py check --strict --metadata --restructuredtext
     check-manifest {toxinidir}
-    black --check
+    black --check .
```

```suggestion
    black --check {toxinidir}
```

While `changedir` defaults to `{toxinidir}` they aren't the same.

https://tox.readthedocs.io/en/latest/config.html#conf-changedir